### PR TITLE
Update References to Ubuntu 18.04 to 20.04

### PR DESCRIPTION
**Pull request by nharper285**
## Summary of the Pull Request

_What is this about?_
We will be deprecating OneFuzz's use of 18.04 Ubuntu Images. This is in accordance with Github's similar deprecation plan - linked [here](https://github.com/actions/runner-images/issues/6002).
This upgrade will prevent OneFuzz build failures when Github's plan goes into effect.

## PR Checklist
* [ ] Applies to work item: #2532 
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

**Automation**
* [ ] ci.yml - build automation updates
* [ ] deploy-onefuzz.yml 

**Documentation**
* [ ] custom-images.md
* [ ] getting-started.md
* [ ] fuzzing-dotnet-with-libfuzzer.md
* [ ] support-platforms.md
* [ ] webhooks_events.md
* [ ] generate_docs.py

**Service Code**
* [ ] models.cs
* [ ] models.py
* [ ] test_models.py

## Validation Steps Performed

_How does someone test & validate?_
* [ ] Ensure Build Works
* [ ] Ensure Deployment Works
* [ ] Integration Tests

### CODE REVIEW

File: docs/custom-images.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ If you have a custom Windows VHD, you should follow the
 [Guide to prepare a VHD for Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/prepare-for-upload-vhd-image).
 
 From there, rather than using Image SKUs such as
-`Canonical:UbuntuServer:18.04-LTS:latest`, use the full resource ID to the
+`Canonical:UbCanonical:0001-com-ubuntu-server-focal:20_04-lts:latest`, use the full resource ID to the
```
Comment (mgreisen) [inline docs/custom-images.md:16]:
```suggestion
`Canonical:0001-com-ubuntu-server-focal:20_04-lts:latest`, use the full resource ID to the
```

Comment (mgreisen) [reply docs/custom-images.md:16]: I think that this image string needs to be updated throughout.

**Review by mgreisen — CHANGES_REQUESTED** [review]

File: docs/custom-images.md (hunk 1 above)
Comment (mgreisen) [reply docs/custom-images.md:16]: This change still needs to be applied.

**Comment by mgreisen** [comment]
We are planning to upgrade all the scalesets to 20.04 before deploying the 20.04 agent.

**Review by Porges — APPROVED** [review]

**Pull request merged**